django-backend/fecfiler/transactions/models.py

```diff
@@ -68,6 +68,7 @@ def form_type(self, value):
     entity_type = models.TextField(null=True, blank=True)
     memo_code = models.BooleanField(null=True, blank=True, default=False)
     force_itemized = models.BooleanField(null=True, blank=True)
```

Comment:
think you need to add force_itemized to the transaction serializer.  it's not getting saved when we toggle it on the front end

Reply:
I thought about that but then I was able to save without any issues on my local. Were you not able to update locally? I can retry and verify.

Reply:
Not needed because we don't do anything fancy with it in the serializer so it gets pulled in with the .get_fields